[stdin]
[stdin]:1
with(this){__filename}
^^^^

SyntaxError: Strict mode code may not include a with statement
    at new Script (vm.js:*)
    at createScript (vm.js:*)
    at Object.runInThisContext (vm.js:*)
    at internal/process/execution.js:*:*
    at [stdin]-wrapper:*:*
    at evalScript (internal/process/execution.js:*:*)
    at internal/main/eval_stdin.js:*:*
    at Socket.<anonymous> (internal/process/execution.js:*:*)
    at Socket.emit (events.js:*:*)
    at endReadableNT (_stream_readable.js:*:*)
42
42
[stdin]:1
throw new Error("hello")
^

Error: hello
    at [stdin]:1:7
    at Script.runInThisContext (vm.js:*)
    at Object.runInThisContext (vm.js:*)
    at internal/process/execution.js:*:*
    at [stdin]-wrapper:*:*
    at evalScript (internal/process/execution.js:*:*)
    at internal/main/eval_stdin.js:*:*
    at Socket.<anonymous> (internal/process/execution.js:*:*)
    at Socket.emit (events.js:*:*)
    at endReadableNT (_stream_readable.js:*:*)
[stdin]:1
throw new Error("hello")
^

Error: hello
    at [stdin]:1:*
    at Script.runInThisContext (vm.js:*)
    at Object.runInThisContext (vm.js:*)
    at internal/process/execution.js:*:*
    at [stdin]-wrapper:*:*
    at evalScript (internal/process/execution.js:*:*)
    at internal/main/eval_stdin.js:*:*
    at Socket.<anonymous> (internal/process/execution.js:*:*)
    at Socket.emit (events.js:*:*)
    at endReadableNT (_stream_readable.js:*:*)
100
[stdin]:1
let x = 100; y = x;
               ^

ReferenceError: y is not defined
    at [stdin]:1:16
    at Script.runInThisContext (vm.js:*)
    at Object.runInThisContext (vm.js:*)
    at internal/process/execution.js:*:*
    at [stdin]-wrapper:*:*
    at evalScript (internal/process/execution.js:*:*)
    at internal/main/eval_stdin.js:*:*
    at Socket.<anonymous> (internal/process/execution.js:*:*)
    at Socket.emit (events.js:*:*)
    at endReadableNT (_stream_readable.js:*:*)

[stdin]:1
let ______________________________________________; throw 10
                                                    ^
10
(Use `* --trace-uncaught ...` to show where the exception was thrown)

[stdin]:1
let ______________________________________________; throw 10
                                                    ^
10
(Use `* --trace-uncaught ...` to show where the exception was thrown)
done
